.gitignore

```diff
@@ -0,0 +1,9 @@
+﻿################################################################################
```

Comment:
Doesn't matter to me, I was under the impression after the last meeting I was at we were going to use the gitignore github as a baseline. No biggie.